### DISCUSSION TITLE
problems with `blitz console` command

### DISCUSSION THREAD

**Discussion by sharmapukar217** [blitz-js/blitz#3931] [General] [answered]
when using `src` directory to store all the source code, blitz console is not loading any resolvers

**Comment by sharmapukar217** [comment]
Also while running the queries from console, ctx is undefined

**Comment by siddhsuresh** [comment, marked as answer]
> Also while running the queries from console, ctx is undefined

Hey @sharmapukar217, this is because, for `ctx`, `req` and `res` to be available in tests or in the console, you could try to create a mock session like in this code snippet https://github.com/blitz-js/blitz/issues/2654#issuecomment-908475727.

> when using `src` directory to store all the source code, blitz console is not loading any resolvers

Created PR #3933 to fix this issue.
